mappings/net/minecraft/data/server/tag/TagProvider.mapping

```diff
@@ -37,7 +39,13 @@ CLASS net/minecraft/class_2474 net/minecraft/data/server/tag/TagProvider
 		ARG 2 info
 	METHOD method_49660 (Lnet/minecraft/class_2474$class_8210;Lnet/minecraft/class_2960;)Z
 		ARG 2 id
+	METHOD method_49661 (Ljava/lang/Void;)Lnet/minecraft/class_2474$class_8211;
+		ARG 1 void_
```

Comment:
```suggestion
		ARG 1 v
```
The underscore is a bit ugly, and I think we use single letters for some lambda params elsewhere

Reply:
We can mass-rename later.